railties/test/application/configuration_test.rb

```diff
@@ -1209,6 +1209,15 @@ def create
       assert_equal false, ActionController::Parameters.action_on_unpermitted_parameters
     end
 
+    test "config.action_controller.default_protect_from_forgery is true by default on production" do
```

Comment:
The behaviour here doesn't depend on the environment, so I'd ✂️ "on production" from the test name.